README.md

```diff
@@ -1,16 +1,17 @@
 # Ethereum Improvement Proposals (EIPs)
 
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+**Before you initiate a pull request**, please read the [EIP-1](https://eips.ethereum.org/EIPS/eip-1) process document. Ideas should be thoroughly discussed on [Ethereum Research](https://ethresear.ch/t/read-this-before-posting/8) or [Ethereum Magicians](https://ethereum-magicians.org/) first.
 
-Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards. **Browse all current and draft EIPs on [the official EIP site](https://eips.ethereum.org/).**
+This repository tracks the ongoing improvements to Ethereum. It contains:
```

Comment:
I think "tracks" is a very loose term here and could be understood differently, i.e. "this is the sole place to get information about what is being worked on for Ethereum", which is clearly the wrong impression to give.

Reply:
Remove definite article at 970c2a926c64275528ad2d92bb6ca707196094c4.

Now we tracking improvements instead of tracking (all of) the improvements.